src/test/java/Calculator/CalculatorTest.java

```diff
@@ -0,0 +1,19 @@
+package Calculator;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CalculatorTest {
+
+    @Test
+    public void CalculatorTest(){
+        Calculator calc = new Calculator();
+        double a = 3;
+        double b= 2;
```

Comment:
please use formatter, your code is not formatted

Reply:
Done, added Save Actions plugin.